dataset/dictionary/characters.json5

```
@@ -4616,7 +4675,7 @@
     pronunciationJa: "めしつかい",
     notes: "[アルレッキーノ](/arlecchino/)のコードネーム",
     notesZh: "[阿蕾奇诺](/arlecchino/)的代号",
-    tags: [ "snezhnaya", "character-sub", "title" ],
+    tags: [ "snezhnaya", "fontaine", "character-main", "title" ],
```

Can you also add `enemy-boss` here?

```suggestion
    tags: [ "snezhnaya", "fontaine", "character-main", "enemy-boss", "title" ],
```